### DISCUSSION TITLE
Please help me how to use this repository

### DISCUSSION THREAD

**Discussion by bheesma002** [anuraghazra/github-readme-stats#2510] [Q&A] [answered]
Please help me how to use this repository for my GitHub profile

**Comment by DURGESH716** [comment, marked as answer]
Hey @bheesma002
This repository is used for customizing your GitHub Profile like, GitHub Stats card, Top Languages card, and many more. 
You can use the below code and paste it in your profile's README.md file:

[![Bheesma's GitHub stats](https://github-readme-stats.vercel.app/api?username=bheesma002)](https://github.com/bheesma002/github-readme-stats)

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=bheesma002&layout=compact)](https://github.com/bheesma002/github-readme-stats)

**Reply by bheesma002** [reply]
Thanks for the reply @DURGESH716
